keyboards/cospad/cospad.h

```diff
@@ -125,6 +125,70 @@
     { k50, ___, k52, ___ } \
 }
 
+
+/* 6x4 numpad2 layout
+ * ,-------------------.
+ * | 00 | 01 | 02 | 03 |
+ * |----|----|----|----|
+ * | 10 | 11 | 12 | 13 |
+ * |----|----|----|----|
+ * | 20 | 21 | 22 | 23 |
+ * |----|----|----|----|
+ * | 30 | 31 | 32 | 33 |
+ * |----|----|----|----|
+ * | 40 | 41 | 42 |    |
+ * |----|----|----| 43 |
+ * | 50 | 51 | 52 |    |
+ * `-------------------'
+ */
+#define LAYOUT_numpad2_6x4( \
```

Comment:
It's wordy, but more descriptive and more in line with our current conventions to name this as:

```suggestion
/* 6x4 numpad with split Plus and 0 keys
 * ,-------------------.
 * | 00 | 01 | 02 | 03 |
 * |----|----|----|----|
 * | 10 | 11 | 12 | 13 |
 * |----|----|----|----|
 * | 20 | 21 | 22 | 23 |
 * |----|----|----|----|
 * | 30 | 31 | 32 | 33 |
 * |----|----|----|----|
 * | 40 | 41 | 42 |    |
 * |----|----|----| 43 |
 * | 50 | 51 | 52 |    |
 * `-------------------'
 */
#define LAYOUT_numpad_6x4_split_plus_zero( \
```